src/homepageExperience/components/FeedbackBar.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import {ComponentSize, FlexBox, JustifyContent} from '@influxdata/clockface'
+import {ThumbsDown, ThumbsUp} from './HomepageIcons'
```

Comment:
```suggestion
import {ThumbsDown, ThumbsUp} from 'src/homepageExperience/components/HomepageIcons'
```

just in case we need to move this file around

Reply:
✅